src/utils.h

```diff
@@ -33,49 +33,50 @@ typedef enum {
 extern log_level_t log_level;
 extern char *log_cid;
 extern gboolean use_syslog;
+extern FILE *output_file;
 
 #define _pexit(s) \
 	do { \
-		fprintf(stderr, "[conmon:e]: %s %s\n", s, strerror(errno)); \
+		fprintf(output_file, "[conmon:e]: %s %s\n", s, strerror(errno)); \
```

Comment:
I think we should log a timestamp here as well. I could follow up on that, WDYT?

Reply:
agreed, though I think that should be optional behind a flag to not add the overhead if a user doesn't want it